content/docs/reference/install.md

```diff
@@ -201,15 +201,37 @@ Alternatively, you can install Pulumi manually.
 <div id="installation_post">
 {{% md %}}
 ## Verify your Installation
-After installing, verify everything is in working order by running the `pulumi` CLI:
+
+After installing Pulumi, verify everything is in working order by running the `pulumi` CLI:
 
 ```bash
 $ pulumi version
 v{{< latest-version >}}
 ```
 
+### Pulumi Not Found Error
+
 If you get an error that `pulumi` could not be found, it means your path has not been configured correctly. Verify that your system's `$PATH` contains the directory containing the `pulumi` CLI installed earlier.
 
+### New Version Warning
+
+If a new version of Pulumi is available, the CLI produces the following example warning when running any of the available commands:
+
+```
+warning: A new version of Pulumi is available. To upgrade from version '0.17.26' to '0.17.27', run
```

Comment:
What do you think of including the _actual_ latest version in the example output? Just so the content seems more fresh in the future?

```suggestion
warning: A new version of Pulumi is available. To upgrade from version '0.17.26' to '{{< latest-version >}}', run
```

Reply:
Excellent suggestion -- thank you!